elements/lisk-transactions/src/errors.ts

```diff
@@ -27,7 +26,8 @@ export class TransactionError extends VError {
 		actual?: string | number,
 		expected?: string | number,
 	) {
-		super(message);
+		super();
+		this.message = message;
```

Comment:
i think we can still use
```
super(message)
```
in this case too

Reply:
Tried this but was causing an issue with swagger. The message property was removed if you call `super(message)` here.